src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -865,36 +865,42 @@ You can use this to create context-aware validators.
-This is useful for pageblocks that display data that changes frequently, such as a list of recent records, an update to a chart, or a list of recent comments.
-
-=== Pageblocks With Refresh
-Not all pageblocks have the refresh option enabled. Below is a list of pageblocks that have the refresh option enabled.
-
-* Record List
-* Chart
-* Comment
-* Iframe
-* Metric
-* Progress
-* Record Organizer
-* Report
-* Social Feed
-* Calendar
-
-=== Enabling Pageblock Refresh
-To enable refreshing on a pageblock, you will need to find the refresh option in the *general tab* when you edit the pageblock. 
-image:compose-configuration/page-block-general.png[Pageblock General Tab, width=49%]
-The refresh option is divided into auto-refresh and manual refresh.
-
-==== Auto-refresh
-When set, pageblocks that have this configuration will automatically refresh based on a set interval. To activate auto refresh, you'll want to enter a numerical value into the input above the *Show refresh button* toggle. The lowest interval allowed is 5 seconds. Setting an interval lower than that would make the input default to 5 seconds. To turn auto-refresh off on a block, you only need to set its value to zero.
-
-==== Manual Refresh
-To activate, toggle the *Show refresh button* switch to its on state. When set, pageblocks that have this configuration will have a refresh button in the top right corner of the pageblock. This button will allow the user to manually refresh the pageblock. To turn manual refresh off on a block, you only need to set its value to false, i.e toggle the switch off.
-
-image:compose-configuration/pageblock-refreshicon.png[Refresh Icon on Pageblocks with manual refresh, width=49%]
+== Page Block Refresh
+
+Page blocks have a refresh option to automatically renew their data without having to reload the entire page. 
+
+This is useful for page blocks that display data that changes frequently, such as a list of new leads, an update to a chart, or a list of recent comments.
+
+[IMPORTANT]
+====
+Automation, File, Content, Nylas, Navigation, and Geometry blocks do not have the refresh option.
+====
+
+
+=== Enabling Auto-refresh
+
+When set, page blocks that have this configuration will automatically refresh based on a set interval.
+
+To activate auto refresh, navigate to the page builder, choose a block that supports refresh, and go to its general tab, there you will find the refresh option.
+
+You'll want to enter a numerical value into the input above the *Show refresh button* toggle, click save and close, save the page and you should see the block start to every time your interval elapses.
+
+The lowest interval allowed is 5 seconds. 
+Setting an interval lower than that would make the input default to 5 seconds. 
+To turn auto-refresh off on a block, you need to set its value to 0.
+
+image:compose-configuration/page-block-general.png["Page block general tab", role="data-zoomable"]
+
+=== Enabling Manual Refresh
```

Comment:
same as auto-refresh vs auto refresh

Reply:
I would opine that they are not the same case. I have chosen to hypenate auto-refresh because the long form would really be automatic refresh right? Meanwhile, in English grammar, it is uncommon to see words that start with *manual* get hypenated.
![image](https://user-images.githubusercontent.com/31524469/232789451-78838c5f-92db-4963-8cea-1c70935edc1c.png)

Reply:
Fair enough